### PR TITLE
feat(search-form): provide styles for the search forms - INNO-530

### PR DESCRIPTION
**Pull request by yhuard**
Ready. Please only review the search form itself, not the headers

### CODE REVIEW

**Comment by yhuard** [comment]
To be clear: I've just started a bit the refactoring of `site-headers` and `header-search-bars` component but they are WIP and will be finished in another PR. The only component to review here is `search-forms`.

File: framework/components/ecl-header-search-bars/ecl-header-search-bars.twig [hunk 1]
```diff
@@ -1,19 +1,7 @@
-<section class="header-search-bar" aria-label="Site tools">
-  <div class="header-search-bar__wrapper1 container-fluid">
+<section class="ecl-header-search-bar" aria-label="Site tools">
+  <div class="container-fluid">
     <section class="block-nexteuropa-europa-search clearfix">
-      <div class="block__content">
```
Comment (kalinchernev) [inline framework/components/ecl-header-search-bars/ecl-header-search-bars.twig]: :+1:

File: framework/components/ecl-search-forms/_search-forms.scss [hunk 2]
```diff
@@ -3,73 +3,43 @@
...
-  margin-bottom: 1em;
-  overflow: hidden;
-
-  .search-form__textfield-wrapper {
-    border: 3px solid transparent;
-    display: table-cell;
-    margin-bottom: 0;
-    position: relative;
-  }
+  display: inline-flex;
```
Comment (kalinchernev) [inline framework/components/ecl-search-forms/_search-forms.scss]: :+1:

File: framework/layout/_top-bar.scss [hunk 3]
```diff
@@ -1,69 +1,43 @@
...
+@include ecl-media-breakpoint-up(lg) {
+  .ecl-top-bar {
     float: right;
-    position: relative;
     padding-bottom: 10px;
+    position: relative;
     top: 6px;
+    width: auto;
 
-    & > .region {
```
Comment (kalinchernev) [inline framework/layout/_top-bar.scss]: :+1:

**Comment by kalinchernev** [comment]
search

Adding this current state of search in europa-theme to remind ourselves why wouldn't we do the improved search at this stage.

**Pull request merged**